krabs/krabs/guid.hpp

```diff
@@ -309,4 +309,21 @@ namespace std
 
         return { managed.get() };
     }
+
+    template<>
+    struct std::hash<krabs::guid>
+    {
+        size_t operator()(const krabs::guid& guid) const
+        {
+            const char* guidBytes = (const char*)&guid;
+            size_t h = 2166136261;
```

Comment:
Observation - might be worthwhile to indicate where this magic number came from.